src/cmd/servo.rs

```diff
@@ -116,19 +115,24 @@ pub fn run_reset(args: &ArgsReset) -> Result<()> {
 /// list servo-compatible devices (Servo V4, Servo V4p1, SuzyQable)
 #[argh(subcommand, name = "list")]
 pub struct ArgsList {
-    /// update the cached servo info. It will take a few seconds per servo.
+    /// print additional info as well (takes more time)
     #[argh(switch)]
-    update: bool,
+    extra: bool,
```

Comment:
What does this `extra` actually print out in addition? I tried but I'm not getting any additional information.
Is this for future extension?

Reply:
It seems `args.extra` only changes `ServoList::discover_slow()` or `ServoList::discover()` and that will not change the printing result. I think old `--update` explains better.

Reply:
Since now ServoList is not cached, --update is a bit misleading so I changed it to extra, since it will retrieve extra info.
And the change will be like this:

without extra
```
(workstation) $ lium servo list --json
{
  "devices": [
    {
      "product": "Servo V4p1",
      "serial": "SERVOV4P1-S-2202180988",
      "usb_sysfs_path": "/sys/bus/usb/devices/1-4.5",
      "tty_list": {
        "Atmega UART": "/dev/ttyUSB3",
        "DUT UART": "/dev/ttyUSB2",
        "Firmware update": "/dev/ttyUSB4",
        "I2C": "/dev/ttyUSB1",
        "Servo EC Shell": "/dev/ttyUSB0"
      },
      "cached_info": null
    }
  ]
}
```

with extra
```
(workstation) $ lium servo list --json --extra
Checking SERVOV4P1-S-2202180988
{
  "devices": [
    {
      "product": "Servo V4p1",
      "serial": "SERVOV4P1-S-2202180988",
      "usb_sysfs_path": "/sys/bus/usb/devices/1-4.5",
      "tty_list": {
        "Atmega UART": "/dev/ttyUSB3",
        "DUT UART": "/dev/ttyUSB2",
        "Firmware update": "/dev/ttyUSB4",
        "I2C": "/dev/ttyUSB1",
        "Servo EC Shell": "/dev/ttyUSB0"
      },
      "cached_info": {
        "mac_addr": "88:54:1f:0f:72:fe"
      }
    }
  ]
}
```

cached_info will be cached in the future, but not at this point... (I'll think about the caching feature later)
... okay, let me change it back to --update (to align with the output...)

Reply:
now they are "SlowInfo" and the flag is "--slow"